demisto_sdk/commands/content_graph/parsers/json_content_item.py

```diff
@@ -94,9 +95,14 @@ def marketplaces(self) -> List[MarketplaceVersions]:
     def get_json(self, git_sha: Optional[str]) -> Dict[str, Any]:
         if self.path.is_dir():
             json_files_in_dir = get_files_in_dir(self.path.as_posix(), ["json"], False)
-            if len(json_files_in_dir) != 1:
-                raise NotAContentItemException(
-                    f"Directory {self.path} must have a single JSON file."
-                )
+            if len(json_files_in_dir) > 1:
```

Comment:
I rewrote the logic here

When there is more than one json file it will raise an error except for the case that the list type is json and then I ignore the *_data.json file